### PR TITLE
updating test pipeline 

### PR DESCRIPTION
**Pull request by saurabh-mridul**
PR consists of below changes
- using inline parameters and variables instead of relying on classic variables groups
- Key vault integration to fetch the secrets as pre job
- added pipeline task to create and delete AKS cluster conditionally 
- added parameters IsMultiRegionRequired to add flexibility options during the run
- variables for one time value assignments

### CODE REVIEW

File: JMeterAKSPipeline/JMeterAKSPipeline.yml [hunk 1]
```diff
@@ -1,54 +1,157 @@
...
+          filePath: ./JMeterAKSPipeline/ExecuteInAKS.ps1
+          arguments: '-AKSClusterName $(AKSClusterNameRegion1) -ResourceGroup $(AKSResourceGroup) -SPNClientId $(AKSSPNClientId) -SPNClientSecret $(AKSSPNClientSecret) -Tenant $(Tenant) -Namespace $(Namespace) -JMeterFolderPath ${{ parameters.JmeterFolderPath }} -JMeterFileName ${{ parameters.JmeterFileName }} -CSVFileNames "$(CSVFileNames)"'
+          pwsh: true
+        condition: "succeeded()"
+
+      - task: PublishPipelineArtifact@1
+        displayName: Publish Pipeline Artifact
+        continueOnError: True
+        inputs:
+          path: ${{ parameters.JmeterFolderPath }}/$(AKSClusterNameRegion2)
```
Comment (Shuvajyoti) [inline JMeterAKSPipeline/JMeterAKSPipeline.yml]: Check path, these are artifacts from region 1 .

**Review by Shuvajyoti — COMMENTED** [review]
Please test out Run Performance in Region1/Region 2(ExecuteInAKS.ps1) task when ready.
Please ensure merge with squash commits.

File: JMeterAKSPipeline/JMeterAKSPipeline.yml (hunk 1 above)
Comment (saurabh-mridul) [reply JMeterAKSPipeline/JMeterAKSPipeline.yml]: done

**Review by agrawalamit31 — APPROVED** [review]
Looks Gud

**Review by Shuvajyoti — APPROVED** [review]

**Pull request merged**